pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -1296,6 +1296,9 @@ private void checkRequestTimeout() {
                 break;
             }
             request = requestTimeoutQueue.poll();
+            if (request == null){
```

Comment:
```suggestion
            if (request == null) {
```

I suspect checkstyle can fail here.